POC-OOBE/OOBE-Sandbox/README.md

```diff
@@ -0,0 +1,32 @@
+# PowerToys Out Of Box Experience Proposal and Code
+
+## Overview
+A team of Microsoft Garage interns have created a proposal for the out-of-box experience after the first time install of the PowerToys app, and a proposal for the UI experience of updates.
+
+## 1. Welcome Screen
+
+#### Rationale:
+
+#### Design:
+
+#### Code:
+
+#### Considerations:
+
+## 2. FancyZones Adaptive Sizing Layout Change
+
+#### Rationale: 
+The Settings update in v18 will introduce the "About Feature" side tab. When minimized, the "About Feature" section goes to the bottom of Settings - harder to find when there are lots of settings. We propose moving the "About Feature" section to the top of the Settings when minimized.
```

Comment:
you should be putting future bits in here.  You don't know for sure what version will contain what.  The last sentence should be sufficient.

Reply:
will do